scripts/init.sh

```diff
@@ -86,7 +86,7 @@ start-frequency-instant)
     -lruntime=debug \
     --instant-sealing \
     --wasm-execution=compiled \
-    --execution=wasm \
+    --execution=native \
```

Comment:
Since most of our local testing is done using `start-frequency-instant` I think changing it to always being `native` might be a bit concerning since there might be some problems that only might happen when you ran it as wasm. While there is a low priority this might happen but being able to test something with the exact or close enough configuration as main-net is always valuable.

Reply:
Unfortunately, the native setting is what enables the use of a debugger. That is a much more important  aspect.  Developers can always switch it to wasm if they need to do additional testing.

Reply:
If the vscode config is already set to use `native` I don't see why we need to set it to `native` in the script. Or are we running the debugger outside of vscode?

Reply:
I changed it to native to make it easier to attach to a running node. It's a command line argument and can be easily changed without needing to compile. 

We can change the makefile back to WASM. Developers needing to attach a debugger can easily modify the arguments.